### PR TITLE
FAT-5006

### PR DESCRIPTION
**Pull request by re-jas**
- https://issues.folio.org/browse/FAT-5006
- [Allure C9217](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1933/allure/)

### CODE REVIEW

File: cypress/e2e/users/loans/manual-anonymization-closed-loans.cy.js [hunk 1]
```diff
@@ -19,157 +19,207 @@ import LostItemFeePolicy from '../../../support/fragments/circulation/lost-item-
...
+    InventoryInstance.deleteInstanceViaApi(testData.instanceId);
+    RequestPolicy.deleteViaApi(policyIds.request);
+    LostItemFeePolicy.deleteViaApi(policyIds.lostItemFee);
+    OverdueFinePolicy.deleteViaApi(policyIds.overdueFine);
+    NoticePolicy.deleteViaApi(policyIds.notice);
+    cy.deleteLoanPolicy(policyIds.loan);
+    cy.deleteLoanType(testData.loanTypeId);
+    NewFeeFine.getUserFeesFines(userData.userId).then((userFeesFines) => {
+      const feesFinesData = userFeesFines.accounts;
+      feesFinesData.forEach(({ id }) => {
```
Comment (penfriend) [inline cypress/e2e/users/loans/manual-anonymization-closed-loans.cy.js:207]: feesFinesData.forEach(({ id })  => cy.wrap(feesFinesData).each({ id })

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js [hunk 2]
```diff
@@ -209,37 +215,57 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
+        userBarcode: userData.barcode,
+      });
+    });
 
-        cy.login(userData.username, userData.password);
+    UserLoans.getUserLoansIdViaApi(userData.userId).then((userLoans) => {
+      const loansData = userLoans.loans;
+      const newDueDate = new Date(loansData[0].loanDate);
+      newDueDate.setDate(newDueDate.getDate() - 1);
+      loansData.forEach((loan) => {
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:250]: loansData.forEach((loan)   => cy.wrap(loansData).each((loan)

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js [hunk 3]
```diff
@@ -113,6 +113,12 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
   };
 
+  const findPatron = () => {
+    cy.visit(TopMenu.usersPath);
+    UsersSearchPane.waitLoading();
+    UsersSearchPane.searchByKeywords(userData.barcode);
+  };
+
   before('Preconditions', () => {
     itemsData.itemsWithSeparateInstance.forEach(function (item, index) {
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:123]: itemsData.itemsWithSeparateInstance.forEach => cy.wrap(itemsData.itemsWithSeparateInstance).each

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js [hunk 4]
```diff
@@ -209,37 +215,57 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
-        cy.get('@items').each((item) => {
-          Checkout.checkoutItemViaApi({
-            id: uuid(),
-            itemBarcode: item.barcode,
-            loanDate: moment.utc().format(),
-            servicePointId: testData.userServicePoint.id,
-            userBarcode: userData.barcode,
-          });
-        });
+        cy.login(userData.username, userData.password);
```
Comment (IhorBohdan) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:223]: You can do this in one line
cy.login(user.username, user.password, { path:TopMenu.ordersPath, waiter: Orders.waitLoading });

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-lost-items-removed-after-lost-item-renewed-returned.cy.js [hunk 5]
```diff
@@ -209,37 +215,57 @@ describe('Patron Block: Maximum number of lost items', () => {
...
-        cy.get('@items').each((item) => {
-          Checkout.checkoutItemViaApi({
-            id: uuid(),
-            itemBarcode: item.barcode,
-            loanDate: moment.utc().format(),
-            servicePointId: testData.userServicePoint.id,
-            userBarcode: userData.barcode,
-          });
-        });
+        cy.login(userData.username, userData.password);
```
Comment (IhorBohdan) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-lost-items-removed-after-lost-item-renewed-returned.cy.js:223]: Also here can do this in one line

**Review by IhorBohdan — APPROVED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js (hunk 4 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:223]: unfortunately it won't work. When you go directly to conditions section, it is not displayed properly

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-lost-items-removed-after-lost-item-renewed-returned.cy.js (hunk 5 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-lost-items-removed-after-lost-item-renewed-returned.cy.js:223]: unfortunately it won't work. When you go directly to conditions section, it is not displayed properly

File: cypress/e2e/users/loans/manual-anonymization-closed-loans.cy.js (hunk 1 above)
Comment (re-jas) [reply cypress/e2e/users/loans/manual-anonymization-closed-loans.cy.js]: done

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js]: done

**Pull request merged**